wiki/markdown/query-aggregated-objects.md

```diff
@@ -115,7 +115,7 @@ Examples of criterias:
 By using a query that contains only empty "criteria" it is possible to return
 all aggregated objects from the database. The aggregated objects will be
 returned from specific collection (which name is defined by property
-aggregated.collection.name) that is stored in specific database (which name is
+aggregations.collection.name) that is stored in specific database (which name is
```

Comment:
Same comment here. Just link to configuration.md instead

Reply:
Done